WordPress/src/main/res/values/strings.xml

```diff
@@ -1320,6 +1320,7 @@
     <string name="reader_expandable_tags_view_overflow_indicator_expand_title" translatable="false">%1$s+</string>
     <string name="reader_expandable_tags_view_overflow_indicator_collapse_title">Hide</string>
     <string name="reader_discover_interests_header">You might like</string>
+    <string name="reader_post_details_header_by">By</string>
```

Comment:
We might want to consider renaming this to something like "reader_post_details_header_by_author_name". "By" can have multiple translations in different languages so I think it'd be good to add the context. Wdyt?

Reply:
Fixed in 82e4378